standalone-tools/wss-scan.sh

```diff
@@ -0,0 +1,121 @@
+###### Information ############################################################################
+# Name:          wss-scan.sh
+# Maintainer:    ODFE Infra Team
+# Language:      Shell
+#
+# About:         This script is to scan the odfe distros for vulnerabilities and licenses
+#                It will scan the repositories and send the WhiteSource link to the mail 
+#                of the user. 
+#
+# Prerequisites: Need to set the Java Path
+#                Need to set the recepient mail in wss-scan.config
```

Comment:
Are you saying user have to make a PR to setup email id? As your script does not provide a way for user to enter the email id during runtime.

Plus, even if so, you already have a "Send mail" block in the workflow, why use command line sendmail again in the shell script? The github runner does not have email relay configured in postfix, thus the sendmail command may not work the same as on your local env.

Reply:
The sendmail was designed for the local run. After the user downloads the code to his desktop, the user can define the mail id in the wss-scan.config to get a mail with the scan details. The Send mail block in the workflow was designed for sending mails from the Github actions.

Reply:
Ok that is fine now, I am not aware it can be run locally.